frontend/ui/widget/datetimewidget.lua

```diff
@@ -396,6 +400,20 @@ function DateTimeWidget:createLayout()
     self:refocusWidget()
 end
 
+function DateTimeWidget:addWidget(widget)
```

Comment:
Let's take care of NT users:
https://github.com/koreader/koreader/blob/43c9ce4a9bcb30e055429f68e88ffae8fd9def94/frontend/ui/widget/inputdialog.lua#L461-L462